airflow/providers/google/cloud/sensors/cloud_composer.py

```diff
@@ -117,3 +128,162 @@ def execute_complete(self, context: dict[str, Any], event: dict[str, str] | None
         if self.soft_fail:
             raise AirflowSkipException(message)
         raise AirflowException(message)
+
+
+class CloudComposerDAGRunSensor(BaseSensorOperator):
+    """
+    Check if a DAG run has completed.
+
+    :param project_id: Required. The ID of the Google Cloud project that the service belongs to.
+    :param region: Required. The ID of the Google Cloud region that the service belongs to.
+    :param environment_id: The name of the Composer environment
```

Comment:
Nitpick. Should we put dots at the end or not?

Reply:
Thanks, fixed it